rails/app/controllers/reference_components_controller.rb

```diff
@@ -1,10 +0,0 @@
-require 'component_information'
```

Comment:
removing reference components controller, i moved this into the component controller because it was weird to have it separated out.